src/main/java/cs2103/v15_1j/jimjim/uifeedback/AddFeedback.java

```diff
@@ -16,9 +16,8 @@ public TaskEvent getTaskEvent() {
     }
 	
 	@Override
-	public String execute(MainViewController con) {
-		// TODO Auto-generated method stub
-		return null;
+	public void execute(MainViewController con) {
+		con.showNotification(taskEvent.getName() + " has been added.");
```

Comment:
I think you should surround the name in quotation marks `"` for clarity.

Also, I think it is better to display all the details about the task / event e.g. `"Meeting with Chris" on Wed, 1 Apr 2016 from 3pm to 4pm has been added.` This should be done by implementint the `toString()` methods of Events, FloatingTasks, DeadlineTasks. But it's not super important so I think it's best for you to decide whether you want to do it or not.